src/main/java/org/sonarsource/sonarlint/ls/AnalysisManager.java

```diff
@@ -360,6 +388,11 @@ public AnalysisResultsWrapper analyzeConnected(ProjectBindingWrapper binding, Wo
         String filePath = FileUtils.toSonarQubePath(getFileRelativePath(baseDir, uri));
         ServerIssueTrackerWrapper serverIssueTracker = binding.getServerIssueTracker();
         serverIssueTracker.matchAndTrack(filePath, issues, issueListener, shouldFetchServerIssues);
+        List<ServerIssue> serverIssues = engine.getServerIssues(binding.getBinding(), filePath);
+        taintVulnerabilitiesPerFile.put(uri, serverIssues.stream()
+          .filter(it -> it.ruleKey().contains(SECURITY_REPOSITORY_HINT))
```

Comment:
We did that in SLE and SLI, maybe we should add a method `engine.getUnresolvedTaintVulnerabilities()`?